### PR TITLE
vehicle_part name to object

### PR DESCRIPTION
**Pull request by snipercup**
#### Summary
```SUMMARY: none```

#### Purpose of change
Contributes to #36110 and #36611

#### Describe the solution
Change name to object

#### Describe alternatives you've considered
Not changing

#### Testing
image
image
image

### CODE REVIEW

File: data/json/vehicleparts/lights.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   {
     "id": "aisle_lights",
     "type": "vehicle_part",
-    "name": "aisle lights",
+    "name": { "str": "aisle lights" },
```
Comment (AMurkin) [inline data/json/vehicleparts/lights.json]: `lights` and `harness` should have plural forms.

File: data/json/vehicleparts/lights.json [hunk 2]
```diff
@@ -220,7 +220,7 @@
   {
     "id": "lit_aisle_horizontal",
     "type": "vehicle_part",
-    "name": "aisle with lights",
+    "name": { "str": "aisle with lights" },
```
Comment (ZhilkinSerg) [inline data/json/vehicleparts/lights.json]:
```suggestion
    "name": { "str": "aisle with lights", "str_pl": "aisles with lights" },
```

File: data/json/vehicleparts/lights.json (hunk 1 above)
Comment (ZhilkinSerg) [inline data/json/vehicleparts/lights.json]:
```suggestion
    "name": { "str": "aisle lights", "str_pl": "aisle lights" },
```

File: data/json/vehicleparts/vehicle_parts.json [hunk 3]
```diff
@@ -1851,7 +1851,7 @@
   {
     "type": "vehicle_part",
     "id": "seatbelt_heavyduty",
-    "name": "5-point harness",
+    "name": { "str": "5-point harness" },
```
Comment (ZhilkinSerg) [inline data/json/vehicleparts/vehicle_parts.json]:
```suggestion
    "name": { "str": "5-point harness", "str_pl": "5-point harnesses" },
```

File: data/json/vehicleparts/vehicle_parts.json [hunk 4]
```diff
@@ -2263,7 +2263,7 @@
   {
     "type": "vehicle_part",
     "id": "craft_rig",
-    "name": "FOODCO kitchen buddy",
+    "name": { "str": "FOODCO kitchen buddy" },
```
Comment (ZhilkinSerg) [inline data/json/vehicleparts/vehicle_parts.json]:
```suggestion
    "name": { "str": "FOODCO kitchen buddy", "str_pl": "FOODCO kitchen buddies" },
```

**Comment by snipercup** [comment]
Cannot add plurals am afraid:
 DEBUG    : Error: data/mods//dda/../../json/vehicleparts/vehicle_parts.json: line 1854:50: str_pl not supported here

    "type": "vehicle_part",
    "id": "seatbelt_heavyduty",
    "name": { "str": "5-point harness", "str_pl":
                                                 ^
                                                  "5-point harnesses" },
    "symbol": ",",
    "color": "light_gray",


 FUNCTION : bool main_menu::load_character_tab(bool)
 FILE     : src/main_menu.cpp
 LINE     : 1072

**Comment by AMurkin** [comment]
Sorry for the confusion. It just looks unusual.

**Pull request merged**